cypress/support/ensure-addon-is-ready.ts

```diff
@@ -4,8 +4,10 @@ import { wait } from '../custom/guards';
 
 beforeEach(() => {
   cy.visit('/?path=/story/examples--select');
-  // wait for iframe to be on the page
-  cy.get('#storybook-preview-iframe');
+  // wait for iframe to be on the page and loaded
+  cy.get('#storybook-preview-iframe[data-is-loaded="true"]');
+  // wait for load to complete
+  cy.wait(100);
```

Comment:
It appears that even after loading has completed, there's a short period where the addon panel still registers as "hidden", which breaks the cypress tests. My best guess is that it could be the CSS transition of the example shrinking to reveal the panel? 100ms is the length of that transition, and with it the cypress tests pass

Reply:
Can you add this detail as a comment to the file? It would be good to record this reasoning in the code